plugins/usdc_coinbase/usdc_coinbase_test.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/stretchr/testify/require"
+	"log"
+	"testing"
+)
+
+func TestNewCoinBaseClient(t *testing.T) {
+	client := NewCoinBaseClient(&defaultConfig)
+	prices, err := client.FetchPrice([]string{"USDC-USD"})
+	require.NoError(t, err)
+	log.Println(prices)
```

Comment:
Perhaps we should have a require here that prices is not empty, rather than a println.

Reply:
Resolved in 4d25fa7c661e76eeaac09170f91d4d6d438fd7ea